model.go

```diff
@@ -137,6 +137,7 @@ type ActionSendHTTP struct {
 	Headers      map[string]string `yaml:"headers,omitempty"`
 	Body         string            `yaml:"body,omitempty"`
 	BodyFromFile string            `yaml:"body_from_file,omitempty"`
+	Sleep        time.Duration     `yaml:"sleep,omitempty"`
```

Comment:
add a `Sleep` to the `ActionSendHTTP` so that we can sleep these actions in order to simulate webhooks without having to use the `ActionSleep`